internal/cri/server/restart.go

```diff
@@ -106,6 +107,14 @@ func (c *criService) recover(ctx context.Context) error {
 		metadata := sandboxstore.Metadata{}
 		err := sbx.GetExtension(podsandbox.MetadataKey, &metadata)
 		if err != nil {
+			if errors.Is(err, errdefs.ErrNotFound) {
+				err = c.client.SandboxStore().Delete(ctx, sbx.ID)
+				if err != nil {
+					return fmt.Errorf("failed to delete sandbox from store: %w", err)
```

Comment:
```suggestion
					return fmt.Errorf("failed to delete sandbox %w, in response to failure to retrieve metadata for sandbox: %w", sbx.ID, err)
```

Reply:
should be `failed to delete sandbox %q` I think. `%w` is for formatting the error, not the string sandbox ID.